modules/keyboard.js

```diff
@@ -44,6 +44,7 @@ class Keyboard extends Module {
       if (handlers.length > 0) {
         let range = this.quill.getSelection();
         handlers.forEach((handler) => {
+          if (evt.defaultPrevented) return;
```

Comment:
Could this be moved to the top of the handler for `addEventListener` since evt should be the same for the rest of the function?